src/SubFolder.php

```diff
@@ -78,6 +78,7 @@ public function process(ServerRequestInterface $request, RequestHandlerInterface
 
     public function getBaseUrl(ServerRequestInterface $request): string
     {
+        /** @var array{SCRIPT_FILENAME:string,PHP_SELF?:string,ORIG_SCRIPT_NAME?:string,DOCUMENT_ROOT?:string} $serverParams */
```

Comment:
[No guarantee](https://www.php.net/manual/en/reserved.variables.server.php#refsect1-reserved.variables.server-description) that `SCRIPT_FILENAME` provided.